src/components/LoanCards/KivaClassicBasicLoanCard.vue

```diff
@@ -542,6 +552,13 @@ export default {
 
 				this.$showTipMsg(msg, 'error');
 			});
+			if (this.showActionButton) {
```

Comment:
This check might want to be on the tracking props instead, just in case a future feature uses the action button and someone forgets the tracking info. Empty events would be confusing for data analytics.

Even just `if (this.atcTrackingProps.category)` might help with maintenance.